test/util/FunnelUtils.spec.tsx

```diff
@@ -25,19 +25,25 @@ describe('funnelUtils', () => {
     expect(res).toEqual(mockRes);
   });
   it('<FunnelTrapezoid /> returns a trapezoid shape with no options', () => {
-    const { container } = render(<FunnelTrapezoid option={{}} {...mockProps} isActive={Boolean(true)} />);
+    const { container } = render(
+      <svg width={100} height={100}>
```

Comment:
Interesting. Did these fail before? If not, why change them? If yes, how come we did not notice? 
I believe this is worth comment more on in the PR summary. Please share some more details, if only for knowledge sharing.

Reply:
The pie test was failing because of animation, it was flaky though. I'm running it on a pretty beefy box so I think the tests finished before the default animation time could. 

The others don't fail but they log annoying messages in the console and that could potentially produce the wrong behavior. SVG elements without their container aren't usually able to be used for much

Reply:
Generally I'd keep the two types of changes separate, but this is mostly just laziness :p. If the bug fix was larger/more impactful I would separate things out. There are other test "errors" that have popped up since the last time I worked on things. Seems I should take a a look at those (mostly key related)